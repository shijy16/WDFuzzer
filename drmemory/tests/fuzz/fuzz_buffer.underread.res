# **********************************************************
# Copyright (c) 2015 Google, Inc.  All rights reserved.
# **********************************************************
#
# Dr. Memory: the memory debugger
#
# This library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation;
# version 2.1 of the License, and no later version.
#
# This library is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Library General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public
# License along with this library; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
#
Error #1: UNADDRESSABLE ACCESS beyond heap bounds:
1 threads were executing fuzz targets.
The error thread was executing the fuzz target with input value:
01010000 02000000 03000000 04000000
