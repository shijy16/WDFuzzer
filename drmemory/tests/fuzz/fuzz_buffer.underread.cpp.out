# **********************************************************
# Copyright (c) 2009-2010 VMware, Inc.  All rights reserved.
# **********************************************************
#
# Dr. Memory: the memory debugger
#
# This library is free software; you can redistribute it and/or
# modify it under the terms of the GNU Lesser General Public
# License as published by the Free Software Foundation;
# version 2.1 of the License, and no later version.
#
# This library is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
# Library General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public
# License along with this library; if not, write to the Free Software
# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
#
Buffer: 0x00000000 0x00000002 0x00000003 0x00000004
Buffer: 0x00000101 0x00000002 0x00000003 0x00000004
Buffer: 0x00010001 0x00000002 0x00000003 0x00000004
Buffer: 0x01000001 0x00000002 0x00000003 0x00000004
Buffer: 0x00000001 0x00000003 0x00000003 0x00000004
Buffer: 0x00000001 0x00000102 0x00000003 0x00000004
Buffer: 0x00000001 0x00010002 0x00000003 0x00000004
Buffer: 0x00000001 0x01000002 0x00000003 0x00000004
Buffer: 0x00000001 0x00000002 0x00000002 0x00000004
Buffer: 0x00000001 0x00000002 0x00000103 0x00000004
done
~~Dr.M~~ ERRORS FOUND:
~~Dr.M~~       1 unique,     5 total unaddressable access(es)
~~Dr.M~~       0 unique,     0 total uninitialized access(es)
~~Dr.M~~       0 unique,     0 total invalid heap argument(s)
~~Dr.M~~       0 unique,     0 total warning(s)
~~Dr.M~~       0 unique,     0 total,      0 byte(s) of leak(s)
~~Dr.M~~       0 unique,     0 total,      0 byte(s) of possible leak(s)
